queries.md

```diff
@@ -2,16 +2,28 @@
 
 ## find all customers that live in London. Returns 6 records.
 
+- SELECT CustomerID, CustomerName, City FROM Customers WHERE City = 'London'; 
```

Comment:
Nice work with these queries - I like that you chose relevant information to return for the select statements so the results are readable